<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectomeGenerateDialog</class>
 <widget class="QDialog" name="ConnectomeGenerateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Connectome</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QCheckBox" name="UseFreesurferBox">
     <property name="text">
      <string>Use freesurfer labels names</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="FileNameEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="BrowseButton">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="TractCreator.qrc">
            <normaloff>:/images/Open.png</normaloff>:/images/Open.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="TractCreator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectomeGenerateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectomeGenerateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UseFreesurferBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
